components/BackyardStack.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { AntDesign } from '@expo/vector-icons';
+import { createStackNavigator } from '@react-navigation/stack';
+
+import BackyardView from '../views/BackyardView';
+
+const Stack = createStackNavigator();
+
+const BackyardStack = () => {
+  return (
+    <Stack.Navigator
+      screenOptions={() => ({
+        headerStyle: {
+          backgroundColor: '#111518',
+        },
+        headerTintColor: '#CEC269',
```

Comment:
are't you giving const style  in  the end maybe? coz you did in other files?

Reply:
I don't really see them as opposing methods. If it's not reused and only takes up one line, there's no need for three extra lines in the bottom. IMO I guess :P